tests/common/devices/sonic.py

```diff
@@ -1923,3 +1923,27 @@ def active_ip_interfaces(self, ip_ifs, tbinfo, ns_arg=DEFAULT_NAMESPACE):
                     }
 
         return ip_ifaces
+
+    def show_syslog(self):
+        """
+        Show syslog config
+
+        Args:
+            dut (SonicHost): The target device
+        Return: Syslog config like below
+            [{
+                "server": "2.2.2.2",
+                "source": "1.1.1.1",
+                "port": "514",
+                "vrf": "default",
+              },
+              {
+                "server": "3.3.3.3",
+                "source": "4.4.4.4",
+                "port": "514",
+                "vrf": "mgmt",
+              },
+              ...
+            ]
+        """
+        return self.show_and_parse('sudo show syslog')
```

Comment:
>sudo [](http://example.com/codeflow?start=36&length=4)

Do you really need `sudo`?

Reply:
The command with sudo or no sudo  is ok. Anyway, we can remove it

Reply:
Done